### PR TITLE
Add router

### PR DESCRIPTION
**Pull request by JanekJaniak**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -30,4 +20,13 @@ export default {
...
 
+#nav 
+  padding: 30px
+
+#nav a 
+  font-weight: bold
+  color: rgb(2, 2, 184)
+
+#nav a.router-link-exact-active 
+  color: rgb(11, 158, 91)
```
Comment (jan-gajdowicz) [inline src/App.vue]:
```
.nav
   a
      &.router-link
```

File: src/components/IngredientLossCalc.vue [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<template lang="pug">
+  div.ingredient_loss_ingredient 
+    h1.headin-primary ingredient loss calculator / in progress
```
Comment (jan-gajdowicz) [inline src/components/IngredientLossCalc.vue]: heading-primary

File: src/components/IngredientLossCalc.vue [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<template lang="pug">
+  div.ingredient_loss_ingredient
```
Comment (jan-gajdowicz) [inline src/components/IngredientLossCalc.vue]: classname doesn't make sense

File: src/App.vue [hunk 4]
```diff
@@ -1,23 +1,13 @@
-<template>
-  <div id="app">
-    <HomePage msg="Gastro Helper"/>
-    <MealCostCalculator />
-  </div>
-</template>
+<template lang="pug">
+  div#app
```
Comment (jan-gajdowicz) [inline src/App.vue]: We've decided on using classes

**Review by jan-gajdowicz — CHANGES_REQUESTED** [review]

File: src/components/IngredientLossCalc.vue (hunk 2 above)
Comment (JanekJaniak) [reply src/components/IngredientLossCalc.vue]: :heavy_check_mark:

File: src/components/IngredientLossCalc.vue (hunk 3 above)
Comment (JanekJaniak) [reply src/components/IngredientLossCalc.vue]: :heavy_check_mark:

File: src/App.vue (hunk 4 above)
Comment (JanekJaniak) [reply src/App.vue]: :heavy_check_mark:

File: src/App.vue (hunk 1 above)
Comment (JanekJaniak) [reply src/App.vue]: :heavy_check_mark:

**Review by jan-gajdowicz — APPROVED** [review]

**Pull request merged**